examples/browser/vuejs/connect-two-ae/aepp/package.json

```diff
@@ -13,7 +13,6 @@
   "author": "Enrico Icardi <enrico.icardi@gmail.com>",
   "license": "ISC",
   "dependencies": {
-    "@aeternity/aepp-sdk": "7.4.1",
```

Comment:
Isn't this needed to build the example?

Reply:
The sdk is needed, but I'm proposing to use sdk held in the parent folders as it was before. This makes it possible to release sdk at the same time as examples depending on that. Otherwise, to do the same we have somehow to make a release tag pointing to a commit that contains examples that already use a published sdk from the same tag.